ashlar/filters.py

```diff
@@ -162,35 +163,26 @@ class JsonBFilterBackend(BaseFilterBackend):
     """
     def filter_queryset(self, request, queryset, view):
         """ Filter by configured jsonb_filters on jsonb_filter_field """
+        lookup_name = 'jsonb'
         filter_field = getattr(view, 'jsonb_filter_field', None)
```

Comment:
Not part of this PR, but does this mean that it's not possible to use this filter on models that have more than one JSONB field?